tests/Application/config/routes/setono_sylius_facebook.yaml

```diff
@@ -0,0 +1,2 @@
+setono_sylius_facebook:
+    resource: "@SetonoSyliusFacebookPlugin/Resources/config/routing.yaml"
```

Comment:
We call this `routes` in other plugins, don't we? Should we fix this now that we are having a new major version?

Reply:
Great idea